rotkehlchen/constants/misc.py

```diff
@@ -24,6 +24,8 @@
 KRAKEN_API_VERSION = '0'
 BINANCE_BASE_URL = 'binance.com/'
 BINANCEUS_BASE_URL = 'binance.us/'
+BINANCE_SETTING_SEPARATOR = ' '
```

Comment:
move these constants either inside binance only or in a constants file in a binance subdir if they are only used there